.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: java
+
```

Comment:
We are at the moment integrating directly with bamboo ci and haven't included travis on our architecture

Reply:
ohh thanks for your feedback